serde-support/src/main/java/io/micronaut/serde/support/deserializers/SimpleObjectDeserializer.java

```diff
@@ -38,20 +37,28 @@ final class SimpleObjectDeserializer implements Deserializer<Object>, UpdatingDe
     private final boolean strictNullable;
     private final BeanIntrospection<Object> introspection;
     private final PropertiesBag<Object> properties;
+    private final SerdeDeserializationPreInstantiateCallback preInstantiateCallback;
 
-    SimpleObjectDeserializer(boolean ignoreUnknown, boolean strictNullable, DeserBean<? super Object> deserBean) {
+    SimpleObjectDeserializer(boolean ignoreUnknown, boolean strictNullable,
+                             DeserBean<? super Object> deserBean,
+                             SerdeDeserializationPreInstantiateCallback preInstantiateCallback) {
```

Comment:
this should be annotated with `@Nullable`

Reply:
resolved via https://github.com/micronaut-projects/micronaut-serialization/pull/490/commits/b23dbabcfe3756e461a68fbea7db17dc92a25598